### PR TITLE
Add 3 missing tzs and add support for friendly timezone names

### PR DESCRIPTION
**Pull request by bengotow**
Calendar sync encountered many exceptions for my bengotow@gmail.com Google calendar because it couldn’t find timezones for some of my events. This diff adds a bunch of TZs I discovered in my mail that were missing from https://github.com/regebro/tzlocal.

It also adds support for many more timezone strings, drawn from:
https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
http://micc.mitel.com/kb/Attachment442.aspx

This fixes my calendar.

### CODE REVIEW

**Comment by spang** [comment]
This change LGTM though I'll defer to Karim! Thanks for these fixes!

File: inbox/events/timezones.py [hunk 1]
```diff
@@ -107,11 +110,160 @@
     'W. Europe Standard Time': 'Europe/Berlin',
     'West Asia Standard Time': 'Asia/Tashkent',
     'West Pacific Standard Time': 'Pacific/Port_Moresby',
-    'Yakutsk Standard Time': 'Asia/Yakutsk',
+    'Yakutsk Standard Time': 'Asia/Yakutsk'
 }
 
+# Also add the friendly timezone names
+
+utc_friendly_timezones = {
```
Comment (khamidou) [inline inbox/events/timezones.py]: Caps on UTC?

File: inbox/events/timezones.py [hunk 2]
```diff
@@ -107,11 +110,160 @@
...
+    '(UTC-01:00) Cabo Verde Is.': 'Atlantic/Cape_Verde',
+    '(UTC-01:00) Cape Verde Islands': 'Atlantic/Cape_Verde',
+    '(UTC) Casablanca': 'Africa/Casablanca',
+    '(UTC) Coordinated Universal Time': 'Etc/GMT',
+    '(UTC) Greenwich Mean Time: Dublin, Edinburgh, Lisbon, London': 'Etc/GMT',
+    '(UTC) Dublin, Edinburgh, Lisbon, London': 'Europe/London',
+    '(UTC) Monrovia, Reykjavik': 'Atlantic/Reykjavik',
+    '(UTC+01:00) Amsterdam, Berlin, Bern, Rome, Stockholm, Vienna': 'Europe/Berlin',
+    '(UTC+01:00) Belgrade, Bratislava, Budapest, Ljubljana, Prague': 'Europe/Budapest',
+    '(UTC+01:00) Brussels, Copenhagen, Madrid, Paris': 'Europe/Paris',
```
Comment (khamidou) [inline inbox/events/timezones.py]: What about Daylight Savings Time? In Paris, DST is `UTC+01:00` but summer time is `UTC+02:00`.

File: inbox/events/timezones.py (hunk 1 above)
Comment (bengotow) [inline inbox/events/timezones.py]: Hmm leading caps in local vars OK? What's the standard python approach? Really don't care.

**Comment by khamidou** [comment]
Thanks for looking into this! LGTM minus the two comments I've added inline.

File: inbox/events/timezones.py (hunk 2 above)
Comment (bengotow) [inline inbox/events/timezones.py]: Hmm yeah - technically this is casting CET to CST in the US, I think it'd be more accurate to set it to Etc/GMT+1 than to Europe/Paris which includes the DST. Will look at it tomorrow.

File: inbox/events/timezones.py (hunk 1 above)
Comment (khamidou) [inline inbox/events/timezones.py]: Yeah, `UTC_friendly_timezone` is more pythonic.

Comment (mhahnenberg) [inline inbox/events/timezones.py]: Not sure I agree with this, e.g. see `datetime.utcnow()`

**Pull request merged**